packages/docusaurus/templates/classic/pages/s.module.css

```diff
@@ -0,0 +1,176 @@
+:root {
+  --first-screen-main-section-height: calc((100vh - 60px - 60px) * .6);
+  --first-screen-secondary-section-height: calc((100vh - 60px - 60px) * .4);
+}
+
+.header {
+  color: white;
+  padding: 1rem 0;
+  text-align: center;
+  position: relative;
+  overflow: hidden;
+}
+
+.buttons  {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+}
+.buttons iframe {
+  margin-left: 1rem;
+}
+.logo {
+  position: absolute;
+  bottom: -80px;
+  left: 20px;
+  animation-duration: 2s;
+  animation-name: jackInTheBox;
+}
+@media screen and (min-height: 900px) {
```

Comment:
Sorry you had to write so much responsive queries. I'll try to get them into Infima asap so we can use Infima instead of writing them here.